packages/flutter_web_plugins/test/plugin_event_channel_test.dart

```diff
@@ -7,7 +7,7 @@
 @TestOn('chrome') // Uses web-only Flutter SDK
 
 import 'dart:async';
-import 'dart:ui' as ui;
+import 'dart:ui' as ui; // ignore: unused_import
```

Comment:
Can you pease leave a comment explaining why it's needed. It's not clear otherwise.

Reply:
Done!